gen/js/chromestatus-openapi/package.json

```diff
@@ -16,6 +16,6 @@
     "prepare": "npm run build"
   },
   "devDependencies": {
-    "typescript": "^5.8"
+    "typescript": "^4.0 || ^5.0"
```

Comment:
I think you should revert this file.  We upgrade to typescript 5.8 as part of a security warning IIRC, but the openapi code generator does not know to use that version.